src/crimson/osd/pg.cc

```diff
@@ -744,21 +748,12 @@ PG::do_osd_ops_iertr::future<Ret> PG::do_osd_ops_execute(
       return do_osd_ops_iertr::future<Ret>{crimson::ct_error::eagain::make()};
     });
   }), OpsExecuter::osd_op_errorator::all_same_way(
-    [&ox, m, obc, failure_func=std::move(failure_func), this] (const std::error_code& e) mutable {
-    const bool need_reload_obc = ox.has_seen_write();
-    logger().debug(
-      "do_osd_ops_execute: {} - object {} got error {}, {}; need_reload_obc {}",
-      m,
-      obc->obs.oi.soid,
-      e.value(),
-      e.message(),
-      need_reload_obc);
-    return (
-      need_reload_obc ? reload_obc(*obc)
-                      : interruptor::make_interruptible(load_obc_ertr::now())
-    ).safe_then_interruptible([&e, failure_func=std::move(failure_func)] {
+    [rollbacker, failure_func=std::move(failure_func), m, this]
```

Comment:
captured `this` is not used in the continuation.

Reply:
Fixed.